backend/Tasque.Core.WebAPI/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "Host": {
+    "Logo": "http://16.170.225.154/assets/tasque_logo.svg"
```

Comment:
We have two logo files: with white text and with black text. Now you use a white one. For using black write "tasque_logo_dark.svg"

Reply:
done